modules/openapi-generator/src/main/resources/JavaSpring/api.mustache

```diff
@@ -67,6 +67,9 @@ import java.util.concurrent.{{^jdk8}}Callable{{/jdk8}}{{#jdk8}}CompletableFuture
 {{#virtualService}}
 @VirtualService
 {{/virtualService}}
+{{#-}}
+{{>requestMapping}}
+{{/-}}
```

Comment:
The requestMapping is never generated on the API level with this condition.
I'm not sure to understand why using `{{#-}}` condition to generate it.